docs/docs/en/guide/parameter/file-parameter.md

```diff
@@ -0,0 +1,102 @@
+# FILE Parameter
```

Comment:
Need to add the new document to `https://github.com/apache/dolphinscheduler/blob/dev/docs/configs/docsdev.js`

Reply:
Done